style.css

```diff
@@ -429,15 +434,187 @@ contact me section
 }
 
 /*
-hover effects
+desktop version
 */
 
-button:hover {
-  color: #fff;
-  background-color: #6070ff;
+.nav-desktop-list {
+  display: none;
+  width: 100%;
+  list-style: none;
+  justify-content: space-between;
+}
+
+.nav-desktop-list a {
+  text-decoration: none;
+  color: currentColor;
+}
+
+/*
+media queries
+*/
+
+@media (min-width: 768px) {
+  /*
+  navigation bar
+  */
+  .nav-icon-menu {
+    display: none;
+  }
+
+  .nav-my-logo {
+    flex: 1 1 auto;
+    box-sizing: border-box;
+    font-size: 1.5rem;
+  }
+
+  .nav-desktop-list {
+    display: flex;
+    flex: 1 1 auto;
+    width: 100%;
+    box-sizing: border-box;
+  }
+
+  .nav-flex-container {
+    height: 72px;
+  }
+
+  /*
+  header and welcom text
+  */
+
+  .header-text {
+    border-radius: 0 0 0 100px;
+    background-image: url('./img/desktop/Header\ bgdesktop-header-background.svg');
+    background-size: 100%;
+    background-repeat: no-repeat;
+    padding-left: 18%;
+    padding-right: 18%;
+    padding-bottom: 10%;
+  }
+
+  /*
+  Job Cards Section
+  */
+
+  .cards-container {
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .card-element {
+    flex-direction: row;
+    padding: 2%;
+  }
+
+  .card-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .card-element img {
+    flex: 1;
+  }
+
+  .text-box {
+    flex: 1;
+    padding-left: 2%;
+    padding-right: 2%;
+    box-sizing: border-box;
+  }
+
+  .technology-tags {
+    width: 40%;
+  }
+
+  .project-frame {
+    padding-right: 30%;
+    box-sizing: border-box;
+  }
+
+  /* About Myself section */
+
+  .about-me-wrap {
+    border-radius: 0 100px 0 0;
+    flex-direction: row;
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .about-me-wrap div {
+    flex: 1;
+    padding-right: 3%;
+  }
+
+  .skill-section {
+    border-bottom: 2px solid #dfe1e6;
+  }
+
+  .skill-section ul {
+    display: flex;
+    flex-direction: row;
+    gap: 2%;
+    padding-right: 35%;
+  }
+
+  .skill-section li {
+    display: flex;
+    flex-direction: column;
+    align-items: baseline;
+  }
+
+  hr {
+    border: 1px solid black;
+  }
+
+  .getResumeButton {
+    font-size: 1.75rem;
+  }
+
+  /*
+  Desktop Contact me version
+  */
+
+  .contact-me-container {
+    background-image: url(./img/desktop/desktop-footer-bakcground/contact\ form\ background\ shapes.svg);
+    background-size: 100%;
+    padding-left: 29%;
+    padding-right: 29%;
+    padding-top: 10%;
+    padding-bottom: 15%;
+  }
+
+  #contac-form {
+    padding-left: 5%;
+    padding-right: 5%;
+  }
+
+  #get-in-touch {
+    margin-left: 35%;
+  }
```

Comment:
- [ ] I recommend making sure the Contact button matches the Figma design by having proper text boldness, border-radius & padding.

Your design:
![image](https://user-images.githubusercontent.com/86069740/236356670-a6fc8b75-63cf-41c4-8d09-a4c18d2b0b85.png)

Figma:
![image](https://user-images.githubusercontent.com/86069740/236356709-370ff6f0-46c9-4a10-b9a2-c48371ceb885.png)